### PR TITLE
SAMZA-1961: Doc improvements for meetups/blogs, adding abstracts

### PR DESCRIPTION
**Pull request by Sanil15**
@vjagadish please have a look

### CODE REVIEW

File: docs/_blog/2018-09-26-stangeloop-recap.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ excerpt_separator: <!--more-->
    limitations under the License.
 -->
 
-Strange Loop, St. Louis, MO
+Talk By Kartik Paramasivam (Director of Engineering, LinkedIN)
```
Comment (prateekm) [inline docs/_blog/2018-09-26-stangeloop-recap.md]: LinkedIn

File: docs/_meetups/dec-2017.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ meetups:
           image: https://img.youtube.com/vi/YDGIDO29Dqk/0.jpg
       slides:
           url: https://www.slideshare.net/SamarthShetty2/stream-processing-using-samza-sql
+    abstract: Deep Dive into Samza's SQL library, api and architecture
```
Comment (prateekm) [inline docs/_meetups/dec-2017.md]: API

File: docs/_meetups/dec-2017.md [hunk 3]
```diff
@@ -23,6 +24,7 @@ meetups:
           website: 
           image:
           affiliation: LinkedIn
+      abstract: A look into LinkedIn’s Search Engine indexing pipeline, focusing on it's leverage of Kafka and Samza to ingest over 10K events per second of real time updates
```
Comment (prateekm) [inline docs/_meetups/dec-2017.md]: its

File: docs/_meetups/july-2018.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ meetups:
           website: 
           image:
           affiliation: LinkedIn
-      abstract: Apache Beam is an open source unified programming model to define and execute data processing pipelines
+      abstract: Apache Beam is an open source unified programming model to define and execute data processing pipelines, this talks explains how Linkedin build Samza Runner to leverage Beam
```
Comment (prateekm) [inline docs/_meetups/july-2018.md]: LinkedIn

File: docs/_meetups/mar-2018.md [hunk 5]
```diff
@@ -24,6 +24,7 @@ meetups:
           website:
           image:
           affiliation: LinkedIn
+      abstract: Venice is a distributed key-value store called Venice, which specializes in serving the datasets computed in Hadoop and Samza at LinkedIn. This talk covers how Linkedin built Venice by leveraging Kafka and how it empowers new Samza use cases at LinkedIn.
```
Comment (prateekm) [inline docs/_meetups/mar-2018.md]: covers how LinkedIn

File: docs/_blog/2018-09-26-stangeloop-recap.md (hunk 1 above)
Comment (Sanil15) [reply docs/_blog/2018-09-26-stangeloop-recap.md]: ok

File: docs/_meetups/dec-2017.md (hunk 2 above)
Comment (Sanil15) [reply docs/_meetups/dec-2017.md]: ok

File: docs/_meetups/dec-2017.md (hunk 3 above)
Comment (Sanil15) [reply docs/_meetups/dec-2017.md]: ok

File: docs/_meetups/july-2018.md (hunk 4 above)
Comment (Sanil15) [reply docs/_meetups/july-2018.md]: ok

File: docs/_meetups/mar-2018.md (hunk 5 above)
Comment (Sanil15) [reply docs/_meetups/mar-2018.md]: ok

File: docs/_meetups/nov-2016.md [hunk 6]
```diff
@@ -10,17 +10,20 @@ meetups:
           website: 
           image:
           affiliation: LinkedIn
+      abstract: In this presentation Kartik will quickly walk-through the journey of Samza so far, Current state of Samza and cover some of the key differentiators of Samza in the crowded field of Stream processing and whats next in Samza
```
[the remark this answers is not in the document]
Comment (Sanil15) [reply docs/_meetups/nov-2016.md]: ok

**Pull request closed without merging**